lib/bpk-component-map/src/BpkIconMarker.android.js

```diff
@@ -39,7 +39,15 @@ import {
 const dynamicStyles = BpkDynamicStyleSheet.create(androidStyles);
 
 const BpkIconMarker = (props: Props) => {
-  const { disabled, icon, latitude, longitude, onPress, status } = props;
+  const {
+    disabled,
+    icon,
+    latitude,
+    longitude,
+    onPress,
+    status,
+    zIndex,
```

Comment:
Why isn't this just spread with `rest`?

Reply:
I guess because we called it out as a user prop and goes into a specific part of the view.

Unless you mean more generally with the props?

Reply:
Yeah I'm wondering why we weren't spreading props before. Seems we shouldn't have had to change the component to allow this.